context/src/main/java/io/opentelemetry/context/Context.java

```diff
@@ -99,6 +99,23 @@ static Context root() {
     return ArrayBasedContext.root();
   }
 
+  /**
```

Comment:
Maybe worth documenting the difference between this and the other wrap equivalent.

Reply:
Added some more detailed docs